service/src/test/resources/application-plugin.yaml

```diff
@@ -0,0 +1,21 @@
+# A Spring application profile for testing with a local postgres database.
+# The database configuration here should match local-dev/local-postgres-init.sql
+tanagra:
+  jdbc:
+    dataSources:
+      # In a production instance, these values should be set with environment variables,
+      # either directly eg. TANAGRA_JDBC_DATA_SOURCES_0_PASSWORD, or injection eg. ${YOUR_ENV_VARIABLE}
+      - dataSourceId: test-data-source
+        url: jdbc:postgresql://127.0.0.1:5432/tanagra
```

Comment:
If you decide to store access control state in a separate DB from core Tanagra (option 1 in previous comment), then it should have a different URL from core Tanagra's DB.

Reply:
If you're going to use separate DBs, then you should test that. You can modify the script that start/stops a DB to add a new one, if this is the route you choose.

Reply:
The database should be setup and torn down before each test so shouldn't matter what is being used. But I'll create a separate database if that's what you want.

Reply:
Currently, the Postgres DB is setup by a script, not the test code. So I don't think it will be setup/torn down before each test. Is that what you observed?

Reply:
I didn't realize your current tests weren't using setup/teardown.

This datasource was used by the simple access control plugin and probably shouldn't be in this PR since I pulled that out of this PR. I'll remove remove it and add it to the new repo I'm going to have to create because Verily doesn't want Tanagra to have a built-in way to manage access control for multiple users.

Reply:
Removed the ability for Tanagra to hand a datasource to a plugin, so this isn't relevant anymore.

Reply:
I disagree with this statement:
> Verily doesn't want Tanagra to have a built-in way to manage access control for multiple users

This is option 2 that I outlined above. As I said there, I think it would be great to add this functionality to core Tanagra. However, if something is in core Tanagra, then you should just use the same DB/data source that is already defined, and there is no need for separate datasource definitions.